setup.py

```diff
@@ -32,6 +37,7 @@
     tests_require=tests_require,
     package_data={'riak': ['erl_src/*']},
     description='Python client for Riak',
+    long_description=long_description,
```

Comment:
Yep, but you can see it here too:

https://testpypi.python.org/pypi?:action=display&name=riak&version=2.4.2

That's from pushing to the test pypi site with the fix.